zulipterminal/ui_tools/views.py

```diff
@@ -932,15 +932,15 @@ def keypress(self, size: urwid_Size, key: str) -> str:
 
 
 class StreamInfoView(PopUpView):
-    def __init__(self, controller: Any, color: str,
-                 name: str, desc: str) -> None:
-        # Add 4 (for 2 Unicode characters on either side) to the popup title
-        # length to make sure that the title gets displayed even when the
-        # content is shorter than the title length (+4 Unicode characters).
-        self.width = max(len(desc), len("# {}".format(name)) + 4) + 2
```

Comment:
Does this change match the refactoring? Another reason that moving the one-column code  into this commit might make sense is that the `len(desc)` code is replaced by the `max(textwidth, ...)` code? So what we use in the first commit should translate to the changes here.